### ISSUE TITLE
hope the session can be stored in Redis

### ISSUE THREAD

**Issue by goophps** [MasoniteFramework/masonite#673]
### Is your feature request related to a problem?

hope the session can be stored in Redis。In our work, it is stored in redis.

### Describe the solution you'd like

hope the session can be stored in Redis

### Would this be a breaking change ?

- [ ] Yes

**Comment by circulon** [comment]
Hi @goophps 

Yes this is now available in M4 but is not yet fully documented.

Add this driver info to your session.py providers config:
```python
DRIVERS = {
    "default": "redis",
    "redis": {
        "host": "127.0.0.1",
        "port": 6379, # optional
        "password": "", # optional
        "timeout": 60 * 60,  # the session data auto expunge timout in seconds
        "namespace": "something", # this will be the prefix on each piece of session data saved. Can be left empty
        "options": {
           # additional redis module driver config options here
            "db": 1, 
            "decode_responses": True, # note this MUST be included until PR #653 is accepted and a new release tagged
        },
    }
```

This will allow you to save larger chunks of data to the "default" session as per how the docs illustrate.
Session Flash data is also included.  

I had a need for this use case  as part of the AuthenticationMiddleware so that my (logged in) user instance model could include larger pieces of data (like JWT tokens).

Hope this helps

**Comment by girardinsamuel** [comment]
It is indeed not documented yet but available. We will add it to the docs! In the meantime, I guess we can close this issue 😉
